### ISSUE TITLE
Failed to load snippets from 'C:\Users\abneyl.atom\packages\language-jade\snippets\language-jade.cson'

### ISSUE THREAD

**Issue by Arcanemagus** [devongovett/atom-jade#77]
When launching Atom v1.25-beta with this package enabled you are greeted with the following error notification:
```
Failed to load snippets from 'C:\Users\abneyl.atom\packages\language-jade\snippets\language-jade.cson'
C:\Users\abneyl\.atom\packages\language-jade\snippets\language-jade.cson: Duplicate key 'if condition'
```

Opening the Developer Tools Console shows the full stack trace:
```
Error reading snippets file 'C:\Users\abneyl\.atom\packages\language-jade\snippets\language-jade.cson': Error: C:\Users\abneyl\.atom\packages\language-jade\snippets\language-jade.cson: Duplicate key 'if condition'
    at Object.detectDuplicateKeys (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\season\lib\cson.js:297:19)
    at addProperty (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:153:42)
    at Array.reduce (native)
    at Obj (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:145:34)
    at transformNode (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:221:18)
    at Value (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:102:18)
    at transformNode (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:221:18)
    at addProperty (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:152:21)
    at Array.reduce (native)
    at Obj (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:145:34)
    at transformNode (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:221:18)
    at Value (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:102:18)
    at transformNode (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:221:18)
    at Block (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:99:18)
    at transformNode (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:221:18)
    at Object.parse (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\cson-parser\lib\parse.js:227:22)
    at parseObject (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\season\lib\cson.js:46:29)
    at parseContents (C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\season\lib\cson.js:106:22)
    at C:\Users\abneyl\AppData\Local\atom\app-1.25.0-beta1\resources\app\node_modules\season\lib\cson.js:226:30
    at FSReqWrap.oncomplete (fs.js:114:15)
```

I'm guessing older versions of Atom were simply swallowing this error silently.

**Comment by LinusU** [comment]
Fixed by #76

**Comment by LinusU** [comment]
Published as v0.7.3
